auth/src/main/java/com/farsitel/bazaar/storage/connection/StorageResponseHandler.kt

```diff
@@ -15,24 +15,29 @@ object StorageResponseHandler {
     private const val IAL_STATUE_FAILED: Int = -1
     private const val IAL_STATUE_DEVELOPER_ERROR: Int = -2
     private const val IAL_STATUE_ACCOUNT_NOT_EXISTS: Int = -3
+    private const val IAL_STATUE_EXTRA_IS_NULL: Int = -4
```

Comment:
STATUE is wrong (change the above names also)

Reply:
Done